dev-python/discid/discid-1.2.0.recipe

```diff
@@ -0,0 +1,61 @@
+SUMMARY="Python bindings of libdiscid by MusicBrainz"
+DESCRIPTION="Python-discid implements Python bindings for MusicBrainz Libdiscid."
+HOMEPAGE="https://python-discid.readthedocs.io/"
+COPYRIGHT="Johannes Dewender"
+LICENSE="GNU LGPL v2.1"
+REVISION="1"
+SOURCE_URI="https://files.pythonhosted.org/packages/source/d/$portName/$portName-$portVersion.tar.gz"
+CHECKSUM_SHA256="cd9630bc53f5566df92819641040226e290b2047573f2c74a6e92b8eed9e86b9"
+SOURCE_DIR="discid-$portVersion"
+
+ARCHITECTURES="any"
+
+PROVIDES="
+	$portName = $portVersion
+	"
+REQUIRES="
+	haiku
+	libdiscid
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	"
+
+PYTHON_PACKAGES=(python python36)
+PYTHON_VERSIONS=(2.7 3.6)
```

Comment:
3.7 should be added too.

Reply:
Addressed in dd5467b34aeabf3b95af4d1595ac3c1d926bdfe8